wca-regulations.md

```diff
@@ -77,7 +77,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 2e3) Competitors who have no citizenship may compete as "Stateless".
 - 2f) Competitors must obey venue rules and conduct themselves in a considerate manner.
 - 2g) Competitors must remain quiet when inside the designated competition area. Talking is permitted, but must be kept at a reasonable level, and away from competitors who are actively competing.
-    - 2g3) Competitors in a group in progress must not communicate with other competitors in the group about the scrambled states of the group, unless both competitors are not eligible for any further attempts in the group. Penalty: disqualification of the competitor(s) from the event, at the discretion of the WCA Delegate. Exception: For 3x3x3 Fewest Moves, competitors that have finished their attempt may share the scramble sequence with anyone who is not actively competing in the relevant attempt.
+    - 2g3) Competitors in a group in progress must not communicate with other competitors in the group about the scrambled states of the group, unless all competitors involved are not eligible for any further attempts in the group. Penalty: disqualification of the competitor(s) from the event, at the discretion of the WCA Delegate. Exception: For 3x3x3 Fewest Moves, competitors that have finished their attempt may share the scramble sequence with anyone who is not actively competing in the relevant attempt.
```

Comment:
I'm just worried that "involved" may be interpreted as "actively participating in the communication", which wouldn't include other competitors.

Reply:
Perhaps "all such competitors" instead of "all competitors involved" then